src/FSharpx.Extras/ComputationExpressions/Result.fs

```diff
@@ -0,0 +1,76 @@
+namespace FSharpx
+
+module Result =
```

Comment:
This is missing the actual computation expression builder.

Reply:
I've added a CE builder